src/applications/travel-pay/util/useDatadogRum.js

```diff
@@ -0,0 +1,38 @@
+import { useEffect } from 'react';
+import { datadogRum } from '@datadog/browser-rum';
+
+import environment from 'platform/utilities/environment';
+
+const initializeDatadogRum = () => {
+  if (
+    // Prevent RUM from running on local/CI environments.
+    environment.BASE_URL.indexOf('localhost') < 0 &&
+    // Prevent re-initializing the SDK.
+    !window.DD_RUM?.getInitConfiguration() &&
+    !window.Mocha
+  ) {
+    datadogRum.init({
+      applicationId: 'b0f350ef-b7c0-4226-a65b-4f0b83a4b630',
+      clientToken: 'pub296b1699132775cab51c8d528c50e903',
```

Comment:
I would usually expect to see something like this not hardcoded in here like this... maybe 🤔 

Also, hi Kevin. 👋

Reply:
But all my friends are doing it...

Looks to be something that is at least established in other code ([example1](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/claims-status/containers/ClaimsStatusApp.jsx), [example2](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/vaos/utils/useDatadogRum.js), there are more), if not best practice.